packages/markers/src/browser/problem/problem-contribution.ts

```diff
@@ -185,6 +206,13 @@ export class ProblemContribution extends AbstractViewContribution<ProblemWidget>
         }
     }
 
+    /**
+     * Clears all the markers in the problem channel.
```

Comment:
The docs are incorrect, `problem channel` should be replaced by `problems view`.
The following is also acceptable:

```
Clears all markers.
```

I'm not sure the method is necessary at all in fact.